docs/maintainers/adr/00001-add-implicit-radios-helper.md

```diff
@@ -0,0 +1,115 @@
+# Add helper method to lift Play form Field values in GovukRadios
+
+* Status: accepted
+* Date: 2021-02-12
+
+Technical Story: PLATUI-296
+
+## Context and Problem Statement
+
+- `play-fronted-govuk` library is not Play Framework aware
+- Adding helper methods to the library to utilise Play Framework features will reduce repetition in service code
+
+## Decision Drivers <!-- optional -->
+- Reducing repetition and boilerplate is not only good practice generally, but will important in helping service teams
+  migrate off older frontend libraries to `play-frontend-govuk` and `play-frontend-hmrc`
+- `play-frontend-govuk` is intended to be a direct port of `govuk-frontend`, and so helpers should live in an
+appropriate helper class alongside the view models
+
```

Comment:
possibly add:
- the preference for using implicit classes rather than directly modifying the viewmodel case classes, derived from the govuk-frontend API, for consistency with other helpers, for example the ErrorSummary helpers.
- not replicating the govuk-frontend parameter lists in the helpers, to keep the overhead of upgrading the library low.

Reply:
Added.